internal/sdkprovider/service/account/account_team_member.go

```diff
@@ -76,7 +77,7 @@ func resourceAccountTeamMemberCreate(ctx context.Context, d *schema.ResourceData
 	client := m.(*aiven.Client)
 	accountID := d.Get("account_id").(string)
 	teamID := d.Get("team_id").(string)
-	userEmail := d.Get("user_email").(string)
+	userEmail := strings.ToLower(d.Get("user_email").(string))
```

Comment:
I think we should instead add validation to the `user_email` since we convert email to lowercase without clear communication on behalf of the user. 

Instead we should error during the `plan` operation